src/main/java/pl/sda/model/Location.java

```diff
@@ -19,7 +19,7 @@
     @Column(name = "location_id")
     private Integer id;
 
-    @Column(name = "name", nullable = false)
+    @Column(name = "city_name", nullable = false)
```

Comment:
dlaczego tu zmieniasz nazwe kolumny? co było nie tak z "name" ? Bo ta zmiana juz ma konsekwencje w klasie LocationDAO np, wiec jak nie trzeba zmieniac, to ja bym zostawiła

Reply:
zmieniłem żeby było bardziej czytelne w bazie, możesz przywrócić jeśli powoduje konflikty